### PR TITLE
Testnet fork

### PR DESCRIPTION
**Pull request by Andrew-Pohl**
Fork at block 925000

increase gas target to 15million

### CODE REVIEW

**Comment by Andrew-Pohl** [comment]
Docker image pushed https://hub.docker.com/layers/fusenet/spark-node/2.0.0_OE/images/sha256-663b2b528ca109530c6695c006b3ff4f30070587c839e9cbb8f0735eca7b0c27?context=explore

File: Version_testNet [hunk 1]
```diff
@@ -1,3 +1,3 @@
 DOCKER_IMAGE_FUSE_APP_VERSION="1.0.0"
-DOCKER_IMAGE_FUSE_PARITY_VERSION="1.0.0_OE"
+DOCKER_IMAGE_FUSE_PARITY_VERSION="2.0.0_OE"
```
Comment (leonprou) [inline Version_testNet:2]: this called PARITY but this is OE ><

Comment (Andrew-Pohl) [reply Version_testNet:2]: yeah more to keep backwards compatibility here so you can still update containers by rerunning older quickstarts (using the version file)

File: scripts/quickstart.sh [hunk 2]
```diff
@@ -606,7 +606,7 @@ function run {
         --restart=always \
         $DOCKER_IMAGE_PARITY \
         --role node \
-        --parity-args --no-warp --node-key $NODE_KEY --jsonrpc-threads $NUM_RPC_THREADS --jsonrpc-server-threads $NUM_HTTP_THREADS
```
Comment (leonprou) [inline scripts/quickstart.sh:609]: Why we are changing this?

File: config/spark/spec.json [hunk 3]
```diff
@@ -39,6 +39,13 @@
     "eip145Transition": "0x6bf64",
     "eip1014Transition": "0x6bf64",
     "eip1052Transition": "0x6bf64",
+    "eip1283Transition": "0xcf850",
```
[the remark this answers is not in the document]
Comment (Andrew-Pohl) [reply config/spark/spec.json:42]: Block 850000 😊

File: scripts/quickstart.sh (hunk 2 above)
Comment (Andrew-Pohl) [reply scripts/quickstart.sh:609]: No longer supported in OE

Comment (leonprou) [reply scripts/quickstart.sh:609]: But we are still want to support parity in the quickstart

File: config/spark/spec.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (Andrew-Pohl) [reply config/spark/spec.json:42]: Assuming all validators stay online around 6am UTC on Monday

File: scripts/quickstart.sh (hunk 2 above)
Comment (leonprou) [reply scripts/quickstart.sh:609]: Also need to delete from this place also https://github.com/fuseio/fuse-network/blob/fe2ed591d360fb14dada64c268f85ed686a2fee2/scripts/quickstart.sh#L580 ?

Comment (Andrew-Pohl) [reply scripts/quickstart.sh:609]: Yes this is redundant now

Comment (Andrew-Pohl) [reply scripts/quickstart.sh:609]: fixed at f106776

File: scripts/quickstart.sh [hunk 4]
```diff
@@ -571,7 +571,7 @@ function run {
     "node")
       INSTANCE_NAME=$NODE_KEY
 
-      ## parse parity config
+       ## parse parity config
```
Comment (leonprou) [inline scripts/quickstart.sh:574]: the space here is redundant, no?

**Review by leonprou — APPROVED** [review]

**Pull request merged**